pkg/tempopb/tempo.proto

```diff
@@ -54,6 +54,7 @@ message SearchRequest {
   uint32 end = 6;
   // TraceQL query
   string Query = 8;
+  uint32 SpansPerSpanSet = 9;
```

Comment:
maybe a uint16? i doubt anyone will ever want more than 2^16 spans per spanset. 2^8 is tempting :)